Gemfile

```diff
@@ -2,7 +2,7 @@ source 'http://rubygems.org'
 
 gem 'rails', '~> 4.0.0'
 
-gem 'nzic_models', github: 'NZOI/nzic_models'
+gem 'nzic_models', git: 'https://github.com/NZOI/nzic_models.git'
```

Comment:
Can we upgrade to Bundler 2.0 instead, or at least enable the HTTPS option?

See bundler/bundler#4978

Reply:
> Can we upgrade to Bundler 2.0 instead, or at least enable the HTTPS option?

We should upgrade to Bundler 2.0 eventually, but it's non-trivial because Rails 4.x requires Bundler < 2.0.

The github shortcut will be removed anyway at some point[1], so if we want to use it we should just add the directive

    git_source(:github) { |repo| "https://github.com/#{repo}.git" }

like in the newer Rails and Bundler templates[2][3]. I pushed a commit that does that (dccb9a4).

[1] <a href="https&#58;//&#103;ithub.com/bundler/bundler/issues/5699"><span>bun</span>dler#5699</a> (originally [intended to be removed in 2.0](https://github.com/bundler/bundler/pull/5705/commits/598ab4bef8fefd0a11d8b4b735a6c13b2cfb2e78#diff-ea5f81b03d4209ef02c02845cd95fcbfR461), then [delayed to 3.0](https://github.com/bundler/bundler/pull/7000/commits/8cbbb3985607bbfffa2d26666da163631c4c5acd), and then [delayed further](https://github.com/bundler/bundler/pull/7000/commits/22db310f67c717a972cd5437cf6dcfc4bfc76b26))  
[2] [rails/railties/lib/rails/generators/rails/app/templates/Gemfile.tt](https://github.com/rails/rails/blob/60e19c6d6cb60159af4e2247f29d2cea375a21d6/railties/lib/rails/generators/rails/app/templates/Gemfile.tt)  
[3] [bundler/lib/bundler/templates/Gemfile](https://github.com/bundler/bundler/blob/391b11966efb140a9ba204ca4e05ec7694296ce0/lib/bundler/templates/Gemfile)